geniza/corpus/views.py

```diff
@@ -305,6 +305,15 @@ def get_context_data(self, **kwargs):
                 "page_type": "document",
                 # preload transcription font when appropriate
                 "page_includes_transcriptions": self.object.has_transcription(),
+                "related_documents": [
```

Comment:
probably worth a brief comment so we remember why this is here
```suggestion
                # generate list of related documents that can be filtered by image url for links on excluded images
                "related_documents": [
```

Reply:
Can we only generate if the document has images that are excluded? I don't know if that's something we have easy access to here, but it would prevent an extra call to Solr that is probably not needed for a lot of cases.